src/test/scala/com/raquo/airstream/errors/ObserverErrorSpec.scala

```diff
@@ -301,8 +301,6 @@ class ObserverErrorSpec extends UnitSpec with BeforeAndAfter {
 
     // -- Now check the contramapped observer behaviour
 
-    lowObs.onNext(1)
```

Comment:
Couple things I fixed in tests:
- This was obviously removed accidentally
- Scala 2.12 does not recognize `a => b` as a partial function, it needs `{ case a => b}`

Reply:
Whoopsie my bad. Thanks :)